certbot-apache/certbot_apache/configurator.py

```diff
@@ -822,6 +810,8 @@ def make_vhost_ssl(self, nonssl_vhost):  # pylint: disable=too-many-locals
         # We know the length is one because of the assertion above
         # Create the Vhost object
         ssl_vhost = self._create_vhost(vh_p)
+        if self.conf("handle-sites"):
+            ssl_vhost.enabled = False
```

Comment:
This is always newly generated vhost, and this is the only situation a vhost can actually be in disabled state in the current vhost object lifecycle (we "handle-sites" only for debian based distros). 

However maybe we should instead do the handling logic in vhost object init and populate this value instead. Currently, vhosts are always created with enabled = True.